DropboxMoveFile/index.js

```diff
@@ -0,0 +1,21 @@
+const main = async (inputs, auths, event) => {
+  const fetch = require('isomorphic-fetch');
+  const { Dropbox } = require('dropbox');
+  const dbx = new Dropbox({
+    accessToken: auths.DROPBOX.ACCESS_TOKEN,
+    fetch
+  });
+
+  try {
+    const resp = await dbx.filesMoveV2({
+      fromPath: inputs.fromPath,

```

Comment:
Sorry to guide you wrong. Dropbox module requires inputs with underscore while our system uses camelCase. In this case you will have to change it to.

 from_path: inputs.fromPath,
 to_path: inputs.toPath